### DISCUSSION TITLE
Unable to trigger after 20 minutes

### DISCUSSION THREAD

**Discussion by 99vinod** [quartznet/quartznet#480] [General] [answered]
Hi,

Am using "Quartz.Impl.AdoJobStore.JobStoreTX, Quartz".I had deployed it in IIS and I created  **minute** job.Till 20 minutes the job is running correctly.But after 20minutes the job is unable to trigger.
I hadn't mentioned the enddate also.I had checked all the IIS configurations also.Can you please help me out.

Thank & Regards,
Kumar.

**Comment by lahma** [comment, marked as answer]
IIS has its quirks when it comes to hosting background jobs. See the bottom of the [FAQ ](https://www.quartz-scheduler.net/documentation/faq.html).
